.gitmodules

```diff
@@ -7,3 +7,5 @@
 [submodule "src/config"]
 	path = src/config
 	url = https://git.savannah.gnu.org/git/config.git
+[submodule "src/config/"]
```

Comment:
I found dup module definition at .gitmodules, so it should be removed.
For me, previous src/config repository has an issue which git can't fetch from, so I altered its mirror what yields expected result.
```...url/cgit/... -> ...url/git/....```